### PR TITLE
Update how to use and contributing and uploaded the UI image

### PR DESCRIPTION
**Pull request by Subhanshu0027**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ## LLMinator: Run & Test LLMs
 #### Gradio based tool with integrated chatbot to run & test LLMs directly from HuggingFace. 
 
-A simple tool built on Gradio, LongChain & Torch. 
+A simple tool built on Gradio, LangChain & Torch.
```
Comment (ashish-aesthisia) [inline README.md:4]: Remove this line or merge into line 3 with a better text

File: README.md [hunk 2]
```diff
@@ -10,6 +13,23 @@ A simple tool built on Gradio, LongChain & Torch.
...
+
+To use LLMinator, follow these simple steps:
+
+- Clone the LLMinator repository from GitHub.
+- Navigate to the directory containing the cloned repository.
+- Run the LLMinator tool using the command python llm.py.
+- Access the web interface by opening the provided URL in your browser.
+- Start interacting with the chatbot and experimenting with LLMs!
+
+## 🤝 Contributing
```
Comment (ashish-aesthisia) [inline README.md:26]: Change to Contributions

**Review by ashish-aesthisia — CHANGES_REQUESTED** [review]
Please Update the mentioned changes.

File: README.md (hunk 2 above)
Comment (Subhanshu0027) [reply README.md]: Done with the changes

**Review by ashish-aesthisia — APPROVED** [review]

**Pull request merged**